lnst/RecipeCommon/Perf/Measurements/NeperFlowMeasurement.py

```diff
@@ -200,8 +200,8 @@ def _parse_job_samples(self, job: Job) ->\
         cpu_results = SequentialPerfResult()
 
         if not job.passed:
-            results.append(PerfInterval(0, 0, "transactions", time.time()))
-            cpu_results.append(PerfInterval(0, 0, "cpu_percent", time.time()))
+            results.append(PerfInterval(0, 1, "transactions", time.time()))
+            cpu_results.append(PerfInterval(0, 1, "cpu_percent", time.time()))
```

Comment:
hmmm, i haven't, interesting idea, i'll think about it a bit...

Reply:
resolving this.

i'm still not decided on whether or not it could potentially be useful to create an "invalid interval" with 0 duration...

but either way, it imo doesn't make sense to include it in this PR anyway.